data_structures/binary_tree/binary_tree_mirror.py

```diff
@@ -0,0 +1,44 @@
+# Created by susmith98
+
+
+"""
+Problem Description:
+Given a binary tree, return it's mirror.
+"""
+
+
+def binaryTreeMirrorDict(
+    root: int, binary_tree: dict = {}, binary_tree_mirror: dict = {}
```

Comment:
This is a mutable default parameter https://docs.python-guide.org/writing/gotchas/